src/app/qrcode/page.tsx

```diff
@@ -0,0 +1,70 @@
+'use client';
```

Comment:
```suggestion
'use server';
```
this should be 'use server' instead

Reply:
or please remove 'use client', because 'use server' is default